sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -113,11 +113,6 @@ class AnalysisErrorSuite extends AnalysisTest {
        (ScalarSubquery(testRelation.select('a, dateLit.as('b))) + Literal(1)).as('a)),
        "Scalar subquery must return only one column, but got 2" :: Nil)
 
-  errorTest(
-    "scalar subquery with no column",
-    testRelation.select(ScalarSubquery(LocalRelation()).as('a)),
-    "Scalar subquery must return only one column, but got 0" :: Nil)
```

Comment:
Actually, I think this test update should be done in the other PR. Technically, adding an assert doesn't cause any problem in this test for the current status. Shall we just remove the test changes, and do it in the other PR.